### ISSUE TITLE
Goal: get to 100% code coverage for PaddedMatrices.jl

### ISSUE THREAD

**Issue by DilumAluthge** [chriselrod/PaddedMatrices.jl#13]
For context, here is the current code coverage:

[![Codecov](https://codecov.io/gh/chriselrod/PaddedMatrices.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/chriselrod/PaddedMatrices.jl)

**Comment by chriselrod** [comment]
I agree, but there's also a bug where only the first line of `@avx for` counts as tested when covered, and all the remaining lines count as uncovered.

Probably a LoopVectorization issue, in that it should try and stick the `LineNumberNode`s from the original expression somewhere?

**Comment by DilumAluthge** [comment]
> I agree, but there's also a bug where only the first line of `@avx for` counts as tested when covered, and all the remaining lines count as uncovered.

Yeah I have also noticed this when looking at Codecov for Gaius.jl.

**Comment by DilumAluthge** [comment]
> Probably a LoopVectorization issue, in that it should try and stick the `LineNumberNode`s from the original expression somewhere?

That sounds like it could work.

**Comment by DilumAluthge** [comment]
Issue to track progress on fixing the `@avx` bug: https://github.com/chriselrod/LoopVectorization.jl/issues/178

**Comment by DilumAluthge** [comment]
https://github.com/chriselrod/LoopVectorization.jl/issues/178 has been fixed now (at least, we have a temporary fix)

**Comment by chriselrod** [comment]
And coverage is still 46%.
But that's partly because it needs multithreaded tests.

**Comment by DilumAluthge** [comment]
BTW one of the CI jobs is failing on master b/c the run time was greater than 120 minutes.

**Comment by chriselrod** [comment]
I should've changed that time limit when I added all the threaded tests.
Those tests will just call the single threaded version if there's only a single thread.
